cmd/root.go

```diff
@@ -95,18 +95,6 @@ func pullAndRebase(container string) (imgid string, changed bool) {
 		return
 	}
 
-	// Make sure it has a commit label before pulling
```

Comment:
Maybe we could still require some label to be present?  Dunno that it really matters though.

Reply:
Yeah, I agree. Like a static label that just signals that this is an oscontainer maybe? Then we can error out with a clean error if someone tries to use any other kind of container.